webapp_utils/app.py

```diff
@@ -12,12 +12,13 @@
 
 def create_app():
     """ Creates the application for webapp utils. """
-    utils_app = connexion.App("webapp-utils", options={'swagger_ui': False,
-                                                       'openapi_spec_path': 'openapi.json'})
+    utils_app = connexion.App("webapp-utils", options={'swagger_ui': True,
+                                                       'openapi_spec_path': '/v1/apispec'})
```

Comment:
just keep it `openapi.json` and also change webapp and reposcan to have `openapi.json` as well

there is no need to have `/v1/apispec` anymore